arc/usr/share/asteroid-launcher/watchfaces/Arc.qml

```diff
@@ -0,0 +1,370 @@
+        width: parent.width * 0.95
+        height: width
+        x: (parent.width - width) / 2
+        y: x
+        Rectangle {
+            id: textBg
+            color: 'transparent'
+            anchors.verticalCenter: parent.verticalCenter
+            x: (parent.width - height) / 2
+            height: parent.width  * 0.25
+            width: (watchFace.width + height) / 2
+            layer.enabled: true
+            opacity: 0.32
+            z:arctextbgz
+
+            Rectangle {
+                id: outerCorner
+                color: watchFace.arccolor
+                height: parent.height
+                anchors.verticalCenter: parent.verticalCenter
+                anchors.left: innerCorner.horizontalCenter
+                anchors.right: parent.right
+                radius: height * 0.05
+                z: 1
+            }
+
+            Rectangle {
+                id: innerCorner
+                color: watchFace.arccolor
+                height: parent.height
+                width: height
+                anchors.verticalCenter: parent.verticalCenter
+                anchors.left: parent.left
+                radius: height / 2
+                z:2
+                clip: true
+
+                Rectangle {
+                    width: parent.width
+                    height: parent.height
+
+                    clip: true
+                    rotation: 180
+                    radius: parent.radius
+                    z:2
+                    color: watchFace.arccolor
+
+                    gradient: Gradient {
+                        GradientStop {
+                            position: 0.00
+                            color: watchFace.arccolor
+                        }
+                        GradientStop {
+                            position: Math.max(0.01, (batteryChargePercentage.value/100) - 0.011)
+                            color: watchFace.arccolor
+                        }
+                        GradientStop {
+                            id: batteryGradientStop
+                            position: Math.min(0.99, (batteryChargePercentage.value/100) + 0.011)
+                            color: watchFace.arcbatterylowcolor
+                        }
+                        GradientStop {
+                            position: 1.00
+                            color: watchFace.arcbatterylowcolor
+                        }
+                    }
+                }
+            }
+        }
+
+        Text {
+            id: dateDisplay
+            z: 8
+            font.pixelSize: watchFace.arcdatefontsize
+            color: watchFace.arctextcolor
+            horizontalAlignment: Text.AlignHCenter
+            verticalAlignment: Text.AlignVCenter
+            lineHeight: 0.7
+            anchors.centerIn: parent
+            text: Qt.formatDate(timeBinding, watchFace.arcdateformat)
+            style: Text.Raised
+            styleColor: watchFace.arccolor
+        }
+
+        ListView {
```

Comment:
I have the feeling that this is an overly complex solution :P

Reply:
You feeling might be correct. 
My Goal was to simplify the variant usages (qml files using Arc { } ) and I might have gone too far.
If you think it's worth it, I'll review and refactor when time allows before opening another pull request.

Reply:
Don't worry, I can merge with that. It's just so that you know.
I don't have the time to think of something better either but I'm pretty sure there must be a simpler way.